libs/components/forms/testing/src/checkbox/checkbox-harness.ts

```diff
@@ -117,6 +119,15 @@ export class SkyCheckboxHarness extends SkyComponentHarness {
     }
   }
 
+  /**
+   * Gets the checkbox's hint text.
+   */
+  public async getHintText(): Promise<string> {
+    const hintText = await this.#getHintText();
+
+    return (await hintText?.text())?.trim() ?? '';
```

Comment:
hint text is either set or undefined, so to keep with that response back to consumers should follow the same and return be `string | undefined` and not send back a blank string back.

Reply:
This is the same pattern we are already using for `getHintText()` methods on other harnesses. Empty string is falsy, so consumers can check that it doesn't exist that way.